OG/OG/AST/AntlrToProgramAST.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Generic;
+using OG.AST.MachineSettings;
+namespace OG.gen
+
+{
+    public class AntlrToProgramAST:OGBaseVisitor<string>, ISemanticErrorable
+    {
+        public List<SemanticError> SemanticErrors { get; set; } = new List<SemanticError>();
+        public override string VisitProg(OGParser.ProgContext context)
```

Comment:
Jeg kunne godt tænke mig en AntlrToProgramAST constructor imellem linje 9 og 10, hvor SemanticErrors bliver dependency injected og/eller oprettet hvis den ikke modtager nogen.

Bare for "best practice" skyld :)

Reply:
enig

Reply:
salvom selve listen nok kommer fra program